modules/cluster-logging-collector-tuning.adoc

```diff
@@ -0,0 +1,178 @@
+
+[NOTE]
+====
+You can use these parameters to help determine the trade-offs between latency and throughput. If you want to optimize cluster logging for throughput, you could batch to reduce network packet count; configure larger buffers and queues, delay flushes, and set longer times between retries. To optimize for low latency, you could send data as soon as possible, avoid build-up of batches, have shorter queues and buffers, and use more frequent flush and retries.
+====
+
+These settings are:
+
+* not relevant to most users, default settings should give good general performance
+* only for advanced users with detailed knowledge of Fluentd configuration and performance
+* only for performance tuning, they have no effect on functional aspects of logging
+
+.Advanced Fluentd Configuration Parameters
+[options="header"]
+|===
+
+|Parmeter |Description |Default
+
+|`chunkLimitSize`
+|The maximum size of each chunk. Fluentd stops writing data in a chunk when it reaches this size.
+|`8m`
+
+|`flushAtShutdown`
+|Determine whether to flush all chunks on shutdown. Leave blank for `false`.
+|`true`
+
+|`flushInterval`
+|The interval between chunk flushes. You can use "s" (seconds), "m" (minutes), or "h" (hours).
+|`1s`
+
+|`flushMode`
+a| The method to perform flushes: 
+* `lazy`: flushes chunks based on the `timekey` parameter.
+* `interval`: flushes chunks based on the `flushInterval`.
+* `immediate`: flushes chunks immediately after data is added to a chunk.
+|`interval`
+
+|`flushThreadCount`
+|The number of threads that perform chunk flushing. Increasing the number of threads improves the flush throughput to hide network latency.
+|`2`
+
+|`overflowAction`
+a|The chunking behavior when the queue is full:
+* `throw_exception`: raises an exception to show in the log.
+* `block`: stops data chunking until buffer full issue is resolved.
+* `drop_oldest_chunk`: drop the oldest chunk to accept new incoming chunks.
+|`block`
+
+|`queuedChunkLimitSize` 
+|The number of queued chunks before staged chunks are added to the queue.
+|`32`
+
+|`retryforever`
+|Determine whether to perform flushing indefinitely.
+|`true`
+
+|`retryMaxInterval`
+|The maximum time in seconds for the `exponential_backoff` retry method.
+|`300s`
+
+|`retryType`
+a|The retry type when flushing fails.
+* `exponential_backoff`: increases the time between flush retries. The time doubles each retry until the `retry_max_interval` parameter is reached.
+* `periodic`: retries flushes periodically, based on the `retry_wait` parameter.
+|`exponential_backoff`
+
+|`retryWait`
+|The time in seconds before the next chunk flush.
+|`1s`
+
+|`totalLimitSize`
+|The maximum size of the buffer, which is the total of the stage size and queue size. If the buffer size exceeds this value, Fluentd stop adding data to chunks and fails with an error. All data not in chunks is lost.
+|`8G`
```

Comment:
I suggest to move this options up close to `chunkLimitSize`. These two play together to tune the trade-off between filesystem usage and throughput.